packages/lib/src/vis/line/hooks.ts

```diff
@@ -7,8 +8,8 @@ const CAMERA_FAR = 1000; // R3F's default
 
 export function useCanvasPoints(
   abscissas: number[],
```

Comment:
```suggestion
  abscissas: NumArray,
```

Reply:
I'm planning on updating all the axis types in a future PR, but well spotted :+1: